### PR TITLE
Update venue show action

### PR DESCRIPTION
**Pull request by alexagodzilla**
Hello guys I have just updated the logic inside of the show page to match the current venue with the same longitude and latitude in order display the once we wanna show on the show page

### CODE REVIEW

**Comment by fransan6** [comment]
Morning - thanks for all of this! We don't need `@venue = Venue.find(params[:id])` in the show action as we have the `before_action set_venue` at the top of the file.

As it is now, we have two show actions - should we not just have one?

**Comment by alexagodzilla** [comment]
Hello francesca, yes we definitely need only one show action, I forgot to remove it. It’s definitely a repetition. I might have merged from previous versions and remake it underneath by mistake. Sorry.

File: app/controllers/venues_controller.rb [hunk 1]
```diff
@@ -30,6 +30,26 @@ def create
     end
   end
 
+
+  def show
+    # @booking = @venue.bookings.find { |booking| booking.user == current_user }
```
Comment (jondedman) [inline app/controllers/venues_controller.rb]: @alexagodzilla, Do we need this line in?

Comment (jondedman) [reply app/controllers/venues_controller.rb]: I mean does it need to be uncommented?

**Comment by ilariaperoni** [comment]
LGTM, thank you Ale and Fra

File: app/controllers/venues_controller.rb (hunk 1 above)
Comment (fransan6) [reply app/controllers/venues_controller.rb]: Yes, we could remove that I think

**Pull request merged**